docs/on_failure.md

```diff
@@ -0,0 +1,15 @@
+# Fixing Issues with Discovering A Supported Container Runtime Environment
+
+If you ended up on this page, 
+it seems that either Testcontainers was not able to find a supported container runtime in your environment,
+or you found page while browsing the documentation in search of information on dealing with errors regarding Testcontainers' environment discovery mechanism.
```

Comment:
or you found _this_ page while searching for information to deal with errors regarding the environment discovery mechanism of Testcontainers.